lightctl

```diff
@@ -96,20 +96,20 @@ esac
 if command -v brightnessctl >/dev/null; then
 	prog='brightnessctl'
 	opts="${dev:+-d $dev}"
-	out=$(brightnessctl $opts ${exp:+-e$exp} -m set "${value}%${cmd}")
+	out=$(brightnessctl "$opts" ${exp:+-e$exp} -m set "${value}%${cmd}")
 	light=$(printf '%s\n' "$out" | cut -d, -f4)
 	light=${light%\%}
 
 elif command -v light >/dev/null; then
 	prog='light'
 	opts="${dev:+-s $dev}"
 	case "$cmd" in
-		+) light $opts -A "$value";;
-		-) light $opts -U "$value";;
-		=) light $opts -S "$value";;
+		+) light "$opts" -A "$value";;
```

Comment:
This is wrong, it must not be quoted.

Sorry, I'm not even reading the rest, this looks like yet another case of someone with weak understanding of shell blindly following shellcheck. 🙁

There’s nothing wrong in these scripts.